middlewares.go

```diff
@@ -25,6 +25,13 @@ func HandlerSet() negroni.Handler {
 	})
 }
 
+// TransactionControl is a middleware to handle transactions in pREST
+func TransactionControl() negroni.Handler {
```

Comment:
I don't understand what this function is doing, call `next`? Should she exist?

Reply:
This function do nothing.. Just call next.

Reply:
by what I understood the @fabriziomello  wanted to understand if he is in the right way and now will implement transaction